staging/2018/benchmarks/csv/001_benchmarks_base.csv

```diff
@@ -0,0 +1,1077 @@
+Measure_Name,Measure_ID,Submission_Method,Measure_Type,Benchmark,Decile_3,Decile_4,Decile_5,Decile_6,Decile_7,Decile_8,Decile_9,Decile_10,TOPPED_OUT,
```

Comment:
I think this file should include the 3 lines of headers we talked about

Reply:
And then run `cat staging/2018/benchmarks/csv/001_benchmarks_base.csv | node scripts/benchmarks/parse-benchmarks-data.js 2016 2018` since we want to use this file to gen the json.

Also, Sarah mentioned using the latest CSV attached in the ticket. Wanted to ask if this is it and if you know the difference